tagger.py

```diff
@@ -1,75 +1,33 @@
+import spacy
 import csv
-from nltk.tokenize.treebank import TreebankWordDetokenizer
-#import a file, one sentence at a time
-#reference a dictionary file (This will list each word/part of speech combination) for each word in the sentence
-#tag each word with its POS
-#create new file, one sentence at a time
 
-#Text from 
-#D. Goldhahn, T. Eckart & U. Quasthoff: Building Large Monolingual Dictionaries at the Leipzig Corpora Collection: From 100 to 200 Languages.
-#In: Proceedings of the 8th International Language Resources and Evaluation (LREC'12), 2012
+pathuntagged = input('Enter path to untagged corpus?')
+
+
+FileData = open("/Users/osamaibraheem/Downloads/deu-de_web-wrt_2019_1K/deu-de_web-wrt_2019_1K-sentences.txt", "r")
```

Comment:
Shouldn't we use the value pathuntagged to read the corpus file instead of hardcoded path ?